src/pages/kiman/Main/MainFeed/MainFeed.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useRef } from 'react';
+import AddComment from './CommentFunction/AddComment';
+import './MainFeed.scss';
+
+const MainFeed = ({ mockdata }) => {
+  const inputRef = useRef();
+  const [userComments, setuserComments] = useState(mockdata);
+
+  const [userComment, setuserComment] = useState('');
+
+  const onChange = e => setuserComment(e.target.value);
+
+  const onClick = e => {
+    if (!inputRef.current.value) {
+      alert('댓글을 입력해주세요');
+    } else {
+      setuserComments([
+        ...userComments,
+        {
+          name: 'test1',
+          comment: userComment,
+          id: Math.random(),
+          isLike: false,
+        },
+      ]);
+      inputRef.current.value = null;
```

Comment:
ref 사용할 필요 없이, userComment state만으로도 구현 가능.
```suggestion
      setUserComment('');
```